.dockerignore

```diff
@@ -1,14 +0,0 @@
-# Files that will not be part of COPY to docker image
```

Comment:
File not needed since we don't do COPY any longer in Dockerfile (instead just mount).